docs/src/pages/guides/DB2OnCloudSetup.md

```diff
@@ -164,77 +216,141 @@ The [Db2 driver](https://repo1.maven.org/maven2/com/ibm/db2/jcc/11.5.0.0/jcc-11.
-      --prop-file db2.properties --schema-name FHIRDATA --grant-to FHIRSERVER --pool-size 2
+      --prop-file db2.properties --schema-name FHIRDATA --grant-to FHIRSERVER
     java -jar schema/fhir-persistence-schema-*-cli.jar \
-      --prop-file db2.properties --schema-name FHIRDATA --allocate-tenant default --pool-size 2
+      --prop-file db2.properties --schema-name FHIRDATA --allocate-tenant default
     ```
 
-4. note the tenantKey from the allocate-tenant step above; this will be needed to configure the IBM FHIR Server datasource in the next step
+Note: Save the tenantKey from the `allocate-tenant` step above; this be needed to configure the IBM FHIR Server datasource in the next step
 
 For more information on using the fhir-persistence-schema cli jar, see https://github.com/IBM/FHIR/tree/main/fhir-persistence-schema/README.md.
 
 ### **Configuring an IBM FHIR Server datasource**
 
-The IBM FHIR Server uses a proxy datasource mechanism, allowing new datasources to be added at runtime without requiring a (Liberty Profile) server restart. To configure a FHIR tenant datasource using an API-KEY, use the following template:
+The IBM FHIR Server uses the native Open Liberty datasources. To configure a FHIR tenant datasource for a tenantKey, use the following template in the fhir-server-config.json:
 
 ``` json
         "persistence": {
             "datasources": {
                 "default": {
-                    "tenantKey": "",
+                    "tenantKey": "myTenantKey",
                     "hints" : {
                         "search.reopt": "ONCE"
                     },
                     "type": "db2",
-                    "connectionProperties": {
-                        "serverName": "dashdb-txn-***-************.services.dal.bluemix.net",
-                        "portNumber": 50001,
-                        "databaseName": "BLUDB",
-                        "apiKey": "<API-KEY>",
-                        "securityMechanism": 15,
-                        "pluginName": "IBMIAMauth",
-                        "currentSchema": "FHIRDATA",
-                        "driverType": 4,
-                        "sslConnection": true,
-                        "sslTrustStoreLocation": "resources/security/dbTruststore.jks",
-                        "sslTrustStorePassword": "change-password"
-                    }
                 }
             }
         }
 ```
 
-The persistence configuration is stored in the `fhir-server-config.json` in the tenant and default configuration folders.
-
 Since release 4.3.2 you can use the `search.reopt` query optimizer hint (shown above) to improve the performance of certain search queries involving multiple search parameters. This optimization is currently only available for Db2. Valid values are "ALWAYS" and "ONCE". See Db2 documentation for `REOPT` for more details.
 
+To configure the datasource.xml for db2 create a datasource.xml for the configDropins folder.
+
+#### For IAM User
```

Comment:
again we should present these in a consistent order

Reply:
done